app-leechcraft/lc-advancednotifications/lc-advancednotifications-0.6.75_pre20200510.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit leechcraft
+
+DESCRIPTION="Flexible and customizable notifications framework for LeechCraft"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug"
+
+DEPEND="~app-leechcraft/lc-core-${PV}
+	dev-qt/qtdeclarative:5"
+RDEPEND="${DEPEND}"
+
+pkg_postinst() {
+	einfo "Advanced Notifications supports playing sounds as a notification"
+	einfo "method. Consider installing a media playback plugin to enjoy this"
+	einfo "feature (app-leechcraft/lc-lmp will do, for example)."
+}
```

Comment:
`optfeature` from *eutils.eclass* is better because it can only print the message if the required dependency is not installed.